proc_scripts/align_slice.py

```diff
@@ -1,24 +1,24 @@
 from pyspecdata import *
-def align_and_slice(s, dimension='t2', convwidth=500, threshold=0.05, fl=None):
+def align_and_slice(s, axis='t2', convwidth=500, threshold=0.05, fl=None):
```

Comment:
Here I realized it was more conventional to call this `axis` rather than `dimension`